custom_components/yamaha_ynca/input_helpers.py

```diff
@@ -32,6 +32,7 @@ class Mapping:
     Mapping(ynca.Input.TUNER, ["tun", "dab"]),
     Mapping(ynca.Input.UAW, ["uaw"]),
     Mapping(ynca.Input.USB, ["usb"]),
+    Mapping(ynca.Input.TV, ["tv"]),
```

Comment:
This input needs to be in the section below with an empty list. These mappings link the Input with the "subunit" which is a  way to control stuff related to the input. E.g. netradio subunit allows you to select stations and get metadata from the station. 

There does not seem to be a TV subunit based on the data in the diagnostics provided in #293 

Put it below PHONO to keep it alphabetically.

```
Mapping(ynca.Input.TV, []),
```

Reply:
Adjusting this will probably also fix the failing tests